src/main/java/com/epam/reportportal/listeners/ListenerParameters.java

```diff
@@ -45,6 +45,9 @@
 	private static final int DEFAULT_MAX_CONNECTION_TIME_TO_LIVE_MS = 29900;
 	private static final int DEFAULT_MAX_CONNECTION_IDLE_TIME_MS = 5 * 1000;
 	private static final int DEFAULT_TRANSFER_RETRY_COUNT = 5;
+	private static final boolean DEFAULT_LAUNCH_JOIN_MODE = false;
```

Comment:
Let's have it as `true` by default?
still if it's just 1 thread, nothing changes.
if 2+ it should be changes

Reply:
@HardNorth enable join mode by default?

Reply:
@DzmitryHumianiuk Enabled